src/blog/v8-release-80.md

```diff
@@ -0,0 +1,160 @@
+tags:
+  - release
+description: 'V8 v8.0 .'
+tweet: ''
+---
+
+<!--
+  Hello curious source reader. I'm happy you decided to read my code to see how I did this gag.
+  Unfortunately, it's probably the ugliest code I've ever written. I'm sorry.
+ -->
+
+<div style="position: relative; height: 14em">
+
+<div style="position: absolute; top: -1em; left: -23%; transform: scale(0.8) rotate(-5deg)">
+<blockquote class="twitter-tweet" data-conversation="none" data-lang="en-gb"><p lang="en" dir="ltr">V8 almost at v8</p>&mdash; Josebaba💥 (@fullstackmofo) <a href="https://twitter.com/fullstackmofo/status/1197260632237780994?ref_src=twsrc%5Etfw">20 November 2019</a></blockquote>
+</div>
+
+<div style="position: absolute; top: 2em; left: -5%; transform: scale(0.8) rotate(4deg)">
+<blockquote class="twitter-tweet" data-conversation="none" data-lang="en-gb"><p lang="en" dir="ltr">What happens when v8 reaches v8? 🤔</p>&mdash; Connor &#39;Stryxus&#39; Shearer (@Stryxus) <a href="https://twitter.com/Stryxus/status/1197187677747122176?ref_src=twsrc%5Etfw">20 November 2019</a></blockquote>
+</div>
+
+<div style="position: absolute; top: -3em; left: 27%; transform: scale(0.8) rotate(-2deg)">
+<blockquote  class="twitter-tweet" data-conversation="none" data-lang="en-gb"><p lang="en" dir="ltr">Wait. What happens when we get V8 V8?</p>&mdash; Thibault Molleman (@thibaultmol) <a href="https://twitter.com/thibaultmol/status/1141656354169470976?ref_src=twsrc%5Etfw">20 June 2019</a></blockquote>
+</div>
+
+<div style="position: absolute; top: 3em; left: 45%; transform: scale(0.8) rotate(-1deg)">
+<blockquote class="twitter-tweet" data-conversation="none" data-lang="en-gb"><p lang="en" dir="ltr">Anything special planned for v8 v8.0? 😅</p>&mdash; Greg Miernicki aka 🅖🅡🅔🅖 (@gregulatore) <a href="https://twitter.com/gregulatore/status/1161302336314191872?ref_src=twsrc%5Etfw">13 August 2019</a></blockquote>
+</div>
+
+<div style="position: absolute; top: -3em; left: 62%; transform: scale(0.8) rotate(10deg)">
+<blockquote class="twitter-tweet" data-conversation="none" data-lang="en-gb"><p lang="en" dir="ltr">are you going to be having an extra special party when V8 goes v8 ?</p>&mdash; SignpostMarv (@SignpostMarv) <a href="https://twitter.com/SignpostMarv/status/1177603910288203782?ref_src=twsrc%5Etfw">27 September 2019</a></blockquote>
+</div>
+
+</div>
+
+It’s finally here. Every V8 release, every six weeks when we branch as part of our [release process](/docs/release-process), the question comes up about what will happen when V8 hits version 8. Will we have a party? Will we ship a new compiler? Will we skip versions 8 and 9 and just stay at an eternal V8 version X? Finally, after [over 10 years](/blog/10-years) of work, on our 100th blog post, we’re pleased to announce our newest branch, [V8 ~~version 8.0~~ V8](https://chromium.googlesource.com/v8/v8.git/+log/branch-heads/8.0), and we can finally answer that question:
+
+It’s bug fixes and performance improvements.
```

Comment:
```suggestion
It’s bug fixes, performance improvements, and cool new features.
```

😃?

Reply:
-1 needs more deadpan. I'd keep it as-is

Reply:
Agree with Mathias, the whole point is that it's phrased as being underwhelming ;)